data/topic-schema/src/main/java/net/corda/schema/Schemas.java

```diff
@@ -248,16 +248,6 @@ private UniquenessChecker() {
         public static final String UNIQUENESS_CHECK_TOPIC = "uniqueness.check";
     }
 
-    /**
-     * Verification message schema.
-     */
-    public static final class Verification {
-        private Verification() {
-        }
-
-        public static final String VERIFICATION_LEDGER_PROCESSOR_TOPIC = "verification.ledger.processor";
```

Comment:
Is there no Topic to remove from the schema for this?

Reply:
Whole file was remobved since it was the only topic there: data/topic-schema/src/main/resources/net/corda/schema/Verification.yaml